tests/pages/api/dumpAPI.js

```diff
@@ -0,0 +1,105 @@
+const { I } = inject();
+const assert = require('assert');
+const request = require('request');
+const fs = require('fs');
+const targz = require("tar.gz");
+const path = require('path');
+const {readdirSync} = require("fs");
+const outputDir= 'tests/output/';
+
+module.exports = {
+  /**
+   * Simulates adding new dashboard with custom panels via inner grafana API to keep test
+   * resistance to UI changes in different Grafans versions.
+   *
```

Comment:
Looks like JS doc is a copy from some another function. Could you please remove it or update with a relevant description.